extensions/quoterism/package.json

```
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quoterism-raycast",
+  "title": "Quoterism",
+  "description": "Get access to the Web's largest collection of human inspiration",
+  "icon": "extension-icon.png",
+  "author": "dmitry_fisenko",
+  "categories": [
+    "Fun",
+    "Media"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Quoterism",
+      "subtitle": "Collection of the most inspiring expressions of mankind",
```

```suggestion
      "title": "Get Quote",
      "subtitle": "Quoterism",
```
Or something like that

---

Thank you for your suggestion!